installation/sql/mysql/base.sql

```diff
@@ -105,7 +105,8 @@ INSERT INTO `#__assets` (`id`, `parent_id`, `lft`, `rgt`, `level`, `name`, `titl
 (85, 18, 120, 121, 2, 'com_modules.module.108', 'Privacy Status', '{}'),
 (86, 18, 122, 123, 2, 'com_modules.module.96', 'Popular Articles', '{}'),
 (87, 18, 124, 125, 2, 'com_modules.module.97', 'Recently Added Articles', '{}'),
-(88, 18, 126, 127, 2, 'com_modules.module.98', 'Logged-in Users', '{}');
+(88, 18, 126, 127, 2, 'com_modules.module.98', 'Logged-in Users', '{}'),
+(89, 8, 161, 162, 2, 'com_content.guidedtours.1', 'COM_GUIDEDTOURS_BASIC_GUIDEDTOUR', '{}');
```

Comment:
```suggestion
(89, 1, 161, 162, 2, 'com_content.guidedtours.1', 'COM_GUIDEDTOURS_BASIC_GUIDEDTOUR', '{}');
```
Parent = 8 and lft 161 and rgt 162 do not fit together. See https://en.wikipedia.org/wiki/Nested_set_model .

Parent 8 would be the record here: https://github.com/joomla-projects/soc21_guided-tour/blob/2b0d332dd89fda7fbff7ff5bdf501af45599d151/installation/sql/mysql/base.sql#L35

But I think you want to append it after this record here, at least the lft and rgt values are like that, and I think that would be right: https://github.com/joomla-projects/soc21_guided-tour/blob/2b0d332dd89fda7fbff7ff5bdf501af45599d151/installation/sql/mysql/base.sql#L61

So the parent should be 1 and not 8.